x/did/client/cli/txDID.go

```diff
@@ -0,0 +1,331 @@
+			if err != nil {
+				return err
+			}
+
+			inBuf := bufio.NewReader(cmd.InOrStdin())
+
+			did, err := types.ParseDID(args[0])
+			if err != nil {
+				return err
+			}
+			verificationMethodID, err := types.ParseVerificationMethodID(args[1], did)
+			if err != nil {
+				return err
+			}
+			// read an input file of DID document
+			doc, err := readDIDDocFrom(args[2])
+			if err != nil {
+				return err
+			}
+			privKey, err := getPrivKeyFromKeyStore(verificationMethodID, inBuf)
+			if err != nil {
+				return err
+			}
+			// For proving that I know the private key. It signs on the DIDDocument.
+			sig, err := signUsingCurrentSeq(clientCtx, did, privKey, doc)
+			if err != nil {
+				return err
+			}
+
+			fromAddress := clientCtx.GetFromAddress()
+			err = cmd.Flags().Set(flags.FlagFrom, fromAddress.String())
+			if err != nil {
+				return err
+			}
+
+			msg := types.NewMsgUpdateDID(did, doc, verificationMethodID, sig, fromAddress.String())
+			if err := msg.ValidateBasic(); err != nil {
+				return err
+			}
+			return tx.GenerateOrBroadcastTxCLI(clientCtx, cmd.Flags(), &msg)
+		},
+	}
+	flags.AddTxFlagsToCmd(cmd)
+	return cmd
+}
+
+func CmdDeactivateDID() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "deactivate-did [did] [key-id]",
+		Short: "Deactivate a DID Document",
+		Args:  cobra.ExactArgs(2),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			clientCtx, err := client.GetClientTxContext(cmd)
+			if err != nil {
+				return err
+			}
+
+			inBuf := bufio.NewReader(cmd.InOrStdin())
+
+			did, err := types.ParseDID(args[0])
+			if err != nil {
+				return err
+			}
+			verificationMethodID, err := types.ParseVerificationMethodID(args[1], did)
+			if err != nil {
+				return err
+			}
+			privKey, err := getPrivKeyFromKeyStore(verificationMethodID, inBuf)
+			if err != nil {
+				return err
+			}
+
+			// For proving that I know the private key. It signs on the DID, not DIDDocument.
+			signableDID := types.SignableDID(did)
+			sig, err := signUsingCurrentSeq(clientCtx, did, privKey, signableDID)
+			if err != nil {
+				return err
+			}
+
+			fromAddress := clientCtx.GetFromAddress()
+			msg := types.NewMsgDeactivateDID(did, verificationMethodID, sig, fromAddress.String())
+			if err := msg.ValidateBasic(); err != nil {
+				return err
+			}
+			return tx.GenerateOrBroadcastTxCLI(clientCtx, cmd.Flags(), &msg)
+		},
+	}
+	flags.AddTxFlagsToCmd(cmd)
+	return cmd
+}
+
+// readBIP39ParamsFrom reads a mnemonic and a bip39 passphrase from the reader in the interactive mode.
+// It returns empty strings in the non-interactive mode, so that they can be auto-generated by crypto.GenSecp256k1PrivKey.
+func readBIP39ParamsFrom(interactive bool, reader *bufio.Reader) (string, string, error) {
+	if !interactive {
+		return "", "", nil
+	}
+
+	// mnemonic can be an empty string
+	mnemonic, err := input.GetString("Enter your BIP39 mnemonic, or hit enter to generate one:", reader)
```

Comment:
In before version, `github.com/cosmos/cosmos-sdk/client/input` set alias. 
So it used to `"github.com/cosmos/cosmos-sdk/client`.
But this version not alias. Required import to `"github.com/cosmos/cosmos-sdk/client/input"`